include/swift/Basic/Malloc.h

```diff
@@ -44,4 +44,4 @@ inline void AlignedFree(void *p) {
   
 }
 
-#endif /* __SWIFT_MALLOC_H__ */
+#endif /* SWIFT_BASIC_MALLOC_H */
```

Comment:
Can you change this to use a C++ style commit ala the LLVM style guide:

http://llvm.org/docs/CodingStandards.html

[btw, thanks for making these changes, this is an iterative process of you learning the style that is used in our project = )]